include/uk/plat/irq.h

```diff
@@ -56,6 +58,13 @@ typedef int (*irq_handler_func_t)(void *);
  */
 int ukplat_irq_register(unsigned long irq, irq_handler_func_t func, void *arg);
 
+struct ukplat_event_irq_data {
+	struct __regs *regs;
+	unsigned long irq;
+};
```

Comment:
Please add comments to the structure.
```suggestion
/** Bla */
struct ukplat_event_irq_data {
	/** Bla */
	struct __regs *regs;
	/** Bla */
	unsigned long irq;
};
```